types/stylelint/index.d.ts

```diff
@@ -849,6 +865,13 @@ declare namespace stylelint {
 		column: number;
 	};
 
+	export type FixCallback = () => void | undefined | never;
+
+	export type FixObject = {
+		apply?: FixCallback;
```

Comment:
@romainmenke does this still need to be bikeshed?
e.g. `mutate`
Are you still hesitating?
Just checking so that we can move on.

Reply:
I am in favor of `apply` :)
It's short and clear.